### PR TITLE
Task/4497 Legal Text Cleanup

### PR DESCRIPTION
**Pull request by tcfos**
## Description
Clean up of legal texts for story "Error Log Sharing", screen "Fehlerbericht senden".

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4497

## Screenshots
Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-11 at 08 56 07

Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-11 at 08 56 19

Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-11 at 08 55 06

Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-11 at 08 57 03

Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-11 at 08 57 15

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings [hunk 1]
```diff
@@ -121,14 +121,14 @@
...
 
-"errorReport_Legal_Consent_Intro" = "By tapping on “Accept and Send”, you consent to the following steps:";
+"errorReport_Legal_SendReports_Subline" = "By tapping on “Accept and Send”, you consent to the following steps:";
 
-"errorReport_Legal_Consent_Bullet1_Header" = "The error report you create will be sent to the RKI for error analysis purposes.";
+"errorReport_Legal_SendReports_Bullet1_Part1" = "The error report you create will be sent to the RKI for error analysis purposes.";
 
-"errorReport_Legal_Consent_Bullet1_Paragraph" = "The RKI will analyse the error report in order to be able to identify and fix possible causes of the errors that occur in your app. If you provide the error report ID to the technical support team, then the RKI will be able to associate the error report with you personally.";
+"errorReport_Legal_SendReports_Bullet1_Part2" = " The RKI will analyse the error report in order to be able to identify and fix possible causes of the errors that occur in your app. If you provide the error report ID to the technical support team, then the RKI will be able to associate the error report with you personally.";
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings:130]: Really want to add that space?
```suggestion
"errorReport_Legal_SendReports_Bullet1_Part2" = "The RKI will analyse the error report in order to be able to identify and fix possible causes of the errors that occur in your app. If you provide the error report ID to the technical support team, then the RKI will be able to associate the error report with you personally.";

```

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.legal.strings [hunk 2]
```diff
@@ -121,14 +121,14 @@
...
+"errorReport_Legal_SendReports_Headline" = "Rıza beyanınız";
 
-"errorReport_Legal_Consent_Intro" = "“Kabul ediyorum”a tıklayarak aşağıdaki adımlara onay vermiş olursunuz:";
+"errorReport_Legal_SendReports_Subline" = "“Kabul ediyorum”a tıklayarak aşağıdaki adımlara onay vermiş olursunuz:";
 
-"errorReport_Legal_Consent_Bullet1_Header" = "Tarafınızdan oluşturulan hata raporu, hata analizinin gerçekleştirilmesi için RKI’ye aktarılır.";
+"errorReport_Legal_SendReports_Bullet1_Part1" = "Tarafınızdan oluşturulan hata raporu, hata analizinin gerçekleştirilmesi için RKI’ye aktarılır.";
 
-"errorReport_Legal_Consent_Bullet1_Paragraph" = "RKI, Uygulamanızda ortaya çıkan hataların olası nedenlerini belirlemek ve bunları gidermek için hata raporunu inceleyecektir. Hata raporu kimlik numarasını teknik destek birimine sağladığınızda, RKI bu hata raporunu size kişisel olarak atayabilir.";
+"errorReport_Legal_SendReports_Bullet1_Part2" = " RKI, Uygulamanızda ortaya çıkan hataların olası nedenlerini belirlemek ve bunları gidermek için hata raporunu inceleyecektir. Hata raporu kimlik numarasını teknik destek birimine sağladığınızda, RKI bu hata raporunu size kişisel olarak atayabilir.";
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.legal.strings:130]: Space?
```suggestion
"errorReport_Legal_SendReports_Bullet1_Part2" = "RKI, Uygulamanızda ortaya çıkan hataların olası nedenlerini belirlemek ve bunları gidermek için hata raporunu inceleyecektir. Hata raporu kimlik numarasını teknik destek birimine sağladığınızda, RKI bu hata raporunu size kişisel olarak atayabilir.";

```

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings (hunk 1 above)
Comment (tcfos) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings:130]: Good point. Changed the implementation in class SendErrorLogsViewModel to add the necessary space.

**Review by whiskey — APPROVED** [review]

**Comment by tcfos** [comment]
obsolete

**Pull request closed without merging**